django/contrib/admin/static/admin/css/base.css

```
@@ -242,10 +242,10 @@ details summary {
 
 blockquote {
     font-size: 0.6875rem;
-    color: #777;
+    color: var(--body-quiet-color);
     margin-left: 2px;
     padding-left: 10px;
-    border-left: 5px solid #ddd;
+    border-left: 5px solid var(--body-quiet-color);
```

```suggestion
    border-left: 5px solid currentColor;
```

Possible refactoring – for this I would have suggested using `currentColor` as then any other styling of the blockquote only has to update `color:`, and both the text and border will adjust.